### DISCUSSION TITLE
Localization support (Idea)

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1871] [Ideas]
It would be nice to be able to set a setting in the appsettings file that would add a marker to identify what values are missing from the Resx file.

The changes needed are to the AppSettings file and LocalizableComponent.cs

**Comment by hishamco** [comment]
As I remember this feature was suggested on the DNX days, but not log the missing values in the file, it throws an exception. If we need to implement this the better way is to move it underneath in the `ResourceStringLocalizer`

We could do in the `LocalizationCompnent` as you mentioned but only the controls that inherits will make use of this feature
